src/PrestaShopBundle/Bridge/AdminController/ControllerConfiguration.php

```diff
@@ -35,8 +35,6 @@
 /**
  * This object holds the configuration of a Controller that is being migrated horizontally.
  * Its properties reflect the properties of a legacy PrestaShop controller.
- *
- * @todo: get rid of public properties and use getters + constructor arguments to secure the contract in dedicated PR.
```

Comment:
We're not going to do this anymore?

Reply:
It hasn't been done yet, but now that the bridge object exists and perform a mapping between legacy field names and configuration fields we'll be able in another PR to replace these public properties with getters/setters with explicit types and maybe some checks if needed